content/md/supplier-onboarder/use-the-product-grid.md

```diff
@@ -0,0 +1,199 @@
+---
+id: products-grid-supplier
+themes: manage-products
+title: Get familiar with the **product grid**
+popular: false
+---
+
+# Where is the product grid?
+
+To access your products: click on `Products`. You will land on the product grid.
+
+This page contains all the information you need to start working on your catalog: your products, your categories, some filters, and locales.
+
+![The product grid with all the information you need](../img/Products_views.png)
+
+In the following sections, you will find out how to set up your work environment.
+
+<!--
+# Locale context
+
+## Select a locale context
+
+The locale selected in the dropdown menu is your working locale. When you move from one locale to another, the following product information is updated in the product grid:
+- The category labels (on the left panel)
+- The attribute labels that can be used as filters (in the filter section on the left panel)
+- The labels of attribute groups
+- The attribute option labels used as values (in the product grid)
+- The completeness
+- The product or attribute values in the product grid
+
+![Select a locale context](../img/Products_locales.png)
+
+::: info
+  1. The locales displayed are those required by the retailer for product enrichment.
+  2. The working locale selected on the product grid page stays the same when you open or edit a product form.
+  3. There are two types of locale, the working locale for your products and the user interface locale, which is the language you chose to work on your Onboarder.
+:::
+-->
+
+# Product grid view
+
+A `Gallery` view of the product grid is available. This view can be used to quickly identify and update products thanks to their image.
+
+With the `Gallery` view, you can see the following properties in the grid:
+- Supplier reference
+- Label: attribute used as "label" in the family
+- Image: attribute used as "main picture" in the family
+- Completeness
+<!--
+- Number of complete variant products over the total number of variant products for a given product model (ex: "2/3" means that 2 variants out of 3 are complete, for the given product model)
+-->
+
+:::info
+Other grid features also apply to this view but are not displayed (channel, locale and category selection, filters, pagination, product selection, etc.)
```

Comment:
Everything looks good, except for this sentence that I don't understand.

Reply:
The gallery view only shows the picture and completeness of products. But you can still use the filters and channel selector for example.